netty/src/main/java/io/grpc/netty/CancelClientStreamCommand.java

```diff
@@ -50,8 +51,9 @@
   CancelClientStreamCommand(NettyClientStream stream, Status reason) {
     this.stream = Preconditions.checkNotNull(stream, "stream");
     Preconditions.checkNotNull(reason);
-    Preconditions.checkArgument(EnumSet.of(CANCELLED, DEADLINE_EXCEEDED).contains(reason.getCode()),
-        "Invalid cancellation reason");
+    Preconditions.checkArgument(EnumSet.of(CANCELLED, DEADLINE_EXCEEDED,
```

Comment:
Should these be the same as AbstractClientStream.CANCEL_REASONS ?  (Specifically, UNKNOWN)